modules/store/src/selector.ts

```diff
@@ -447,12 +483,19 @@ export function createSelector(
 
 export function defaultStateFn(
   state: any,
-  selectors: any[],
+  selectors: Selector<any, any>[] | SelectorWithProps<any, any, any>[],
   props: any,
   memoizedProjector: MemoizedProjection
 ): any {
-  const args = selectors.map(fn => fn(state, props));
-  return memoizedProjector.memoized.apply(null, args);
+  if (props === undefined) {
```

Comment:
This check is added because it would break some tests otherwise.
The tests that would fail are the ones that check what the projector is receiving.
I don't know if these are heavily tested, hence the check (I don't want to break any tests on existing code bases).